@xen-orchestra/vmware-explorer/esxi.mjs

```diff
@@ -59,7 +59,7 @@ export default class Esxi extends EventEmitter {
     await Promise.all(
       Object.values(res).map(async ({ datastore, name }) => {
         await Promise.all(
-          datastore.ManagedObjectReference.map(async ({ $value }) => {
+          datastore.ManagedObjectReference?.map(async ({ $value }) => {
```

Comment:
This will not work:

```js
wait Promise.all(undefined)
// → TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```